### DISCUSSION TITLE
Best practice for company wide Quarkus defaults

### DISCUSSION THREAD

**Discussion by d135-1r43** [quarkusio/quarkus#28739] [Q&A] [answered]
At my company, we are adopting Quarkus for (nearly) all our services. I want to set some nice defaults like parent-pom, maven config, build plans etc. that are mandatory for all developers -- yet I like the great dev experience of code.quarkus.io

For Spring Boot I can host my own company [Spring Initializer](https://start.spring.io/). Is there something similar available for Quarkus? Or is a Maven archetype the best solution?

**Comment by maxandersen** [comment, marked as answer]
You can host your own code.quarkus if you wish.

Also possible to tweak standards by having your own quarkus platform.

Wether that is enough for your usecase I'm not sure. Might need some options for better override ability.

**Reply by karstengresch** [reply]
Just for further clarification: clone (better fork, for own adjustments) https://github.com/quarkusio/code.quarkus.io and follow the instructions in README.md to run it.
